src/main/java/mate/academy/onlinebookstore/dto/shoppingcart/UpdateCartItemRequestDto.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.onlinebookstore.dto.shoppingcart;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+
+public record UpdateCartItemRequestDto(
+        @NotNull
```

Comment:
```suggestion
```

Reply:
The same why @Notnull is not needed here?

Reply:
same, it's a primitive